pymoose/moosemodule.h

```diff
@@ -27,23 +20,6 @@ struct module_state
 };
 
 
-// The endianness check is from:
```

Comment:
MOOSE compiles on 32 bit machines but numerical accuracy is bad. These features are now well handled by `c++11` compliant compiler. Clean up!